pyCFOFiSAX/_tree_iSAX.py

```diff
@@ -491,9 +491,9 @@ def get_width_of_level(self, level: int):
 
     def get_width_of_all_level(self):
         """
-        Fonction permettant de retourner la largeur de tous les niveaux dans une liste sachant root niveau 0
+        Function to return the width of all levels in a list that knows root level 0
```

Comment:
that knows root level 0 -> , considering root level = 0

Reply:
```suggestion
        Function to return the width of all levels in a list that knows root level 0 -> , considering root level = 0
```